ui/src/mixins/dark.js

```diff
@@ -9,7 +9,7 @@ export default {
   computed: {
     isDark () {
       return this.dark === null
```

Comment:
Can't you just do here `return !!this.dark || !!(this.$q  && this.$q.dark.isActive)`
Not sure if `?` (optional chanining) can be used here, is it supported in quasar?